rosa_learning/deploying_application_workshop/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,151 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Deploying the OSToy application with Kubernetes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+Deploy the OSToy application by creating and storing the images for the front-end and back-end microservice containers in an image repository. Then create Kubernetes deployments to deploy the application. 
+
+== Retrieving the login command
+
+. If you are not logged in to the command line interface (CLI), access your cluster with the web console.
```

Comment:
```suggestion
. If you are not logged in to the command-line interface (CLI), access your cluster with the web console.
```

Reply:
Couldn't find exact instructions in the style guide I looked at about the hyphen, but within that style guide they used a hyphen when CLI was mentioned in another example...and seems to be the way it is presented in most of our docs....